.env.local.example

```
@@ -0,0 +1,4 @@
+NEXT_PUBLIC_CONVEX_URL=
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY key should go before the NEXT_PUBLIC_CONVEX_URL key

(UnorderedKey)